app/assets/javascripts/components/markdown-editor.js

```diff
@@ -1,4 +1,4 @@
-//= require vendor/@webcomponents/webcomponentsjs/webcomponents-loader.js
+//= require vendor/@webcomponents/webcomponentsjs/webcomponents-bundle.js
```

Comment:
I know it was like this before and seemed to be included correctly but it seems odd to have this included only in the markdown component ?

Reply:
Makes sense to move it to `application.js` now